packages/vapor/scss/controls/inputs.scss

```diff
@@ -64,7 +64,7 @@ textarea {
         padding: 0;
         vertical-align: top;
         background-color: var(--medium-grey);
-        border: 1px solid var(--input-add-on-color);
+        border: 1px solid darken($medium-grey, 10%);
```

Comment:
medium-grey is not a new color :eyes:

Reply:
Yes, the objective of this user story is to remove CSS variables, no visual change will be done. https://github.com/coveo/react-vapor/pull/1895/files#diff-517a27cf28e30b251089bce6cc3eacd9a2e0a5416c6766e6296f122d565c88e9L260